ts/features/payments/home/components/PaymentsHomeTransactionsList.tsx

```diff
@@ -17,21 +17,31 @@ import { getPaymentsLatestBizEventsTransactionsAction } from "../../bizEventsTra
 import { NoticeListItem } from "../../../../../definitions/pagopa/biz-events/NoticeListItem";
```

Comment:
Since `Layout` from `react-native-reanimated` appears to be deprecated, can you achieve the same behavior using `LinearTransition`?

Reply:
Addressed into 0846111f80f6e42c659a33d49e2aac30a553ff28